src/main/java/com/elytradev/smores/block/BlockGemOre.java

```diff
@@ -64,55 +53,7 @@ protected BlockStateContainer createBlockState() {
 	}
 
 	@Override
-	public int getMetaFromState(IBlockState state) {
-		return state.getValue(GEM).getId();
-	}
-
-	@Override
-	public int damageDropped(IBlockState state) {
-		return getMetaFromState(state);
-	}
-
-	@Override
-	public IBlockState getStateFromMeta(int meta) {
-		return this.getDefaultState().withProperty(GEM, EnumGem.values()[meta]);
-	}
-
-	@Override
-	public void getSubBlocks(CreativeTabs tab, NonNullList<ItemStack> items) {
-		int iterator = 0;
-		for (EnumGem gem : EnumGem.values()) {
-			items.add(new ItemStack(this, 1, iterator));
-			iterator++;
-		}
+	public void getDrops(NonNullList<ItemStack> drops, IBlockAccess world, BlockPos pos, IBlockState state, int fortune) {
+		drops.add(new ItemStack(SmoresItems.gem, 1,  getIndexOfMaterial(state.getValue(GEM))));
```

Comment:
This won't work with the Fortune enchantment; check the `fortune` parameter (see vanilla BlockOre for the math and random params used)

Reply:
Yeah, this is just temporary until I figure out what I want to have happen with the fortune enchantment >3<